### PR TITLE
h5py CrayGNU Python 2.7.11 2015.11

### PR DESCRIPTION
**Pull request by gppezzi**
Requires ~~https://github.com/hpcugent/easybuild-easyblocks/pull/848 and~~ https://github.com/hpcugent/easybuild-easyconfigs/pull/2608

### CODE REVIEW

**Comment by boegel** [comment]
@gppezzi: just add `prebuildopts` like in the existing h5py 2.5.0 easyconfigs?

**Comment by gppezzi** [comment]
@boegel  Done. If only that was available when I created this .eb it would have been easier :P

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-CrayGNU-2015.11-XC-Python-2.7.11-parallel.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+prebuildopts = ' python setup.py configure --mpi  && '
```
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-CrayGNU-2015.11-XC-Python-2.7.11-parallel.eb]: we usually put this below `dependencies`, please move this down?

**Comment by gppezzi** [comment]
@boegel  done

**Comment by gppezzi** [comment]
@boegel removed the -XC suffix

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
dora103 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/5135fd47d0e327dd89ada2afd4d6415c for a full test report.

**Comment by gppezzi** [comment]
@boegel I've fixed the sanity check, now it should be good...

**Comment by boegel** [comment]
@gppezzi can you elaborate a bit on the need to bypass the `import h5py` test? What kind of problem are you running into exactly?

**Comment by gppezzi** [comment]
@boegel it looks like similar to the problem with mpi4py and most of the mpi-enabled python libraries we see on the Cray systems: the import fails because mpi subsystem is unavailable on the login nodes.

Those builds usually pass if performed on compute nodes.

**Comment by boegel** [comment]
@gppezzi not a big fan of the hackish workaround, but I see no better option, and the `sanity_check_paths` is pretty strict, so fine

Let's use the `%(pyver)s` and `%(pyshortver)s` templates though, cfr. https://github.com/gppezzi/easybuild-easyconfigs/pull/7

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
daint101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/9440d652856f16eea55dcae78122172b for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @gppezzi!

**Pull request merged**